### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by altissiana**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```
Comment (DoubleTK) [inline Gemfile]: What is this?

File: lib/menus/main_menu.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            ".colorize(:yellow)
+        puts ""
+        puts ""
+        puts ""
+        show
+    end
+
+    def self.show
+        prompt = TTY::Prompt.new
+        choices = %w(Festivals Artist)
```
Comment (DoubleTK) [inline lib/menus/main_menu.rb]: Shouldn't Artist be Artists?

Comment (altissiana) [reply lib/menus/main_menu.rb]: oops

Comment (altissiana) [reply lib/menus/main_menu.rb]: ooops

**Pull request merged**